android/app/src/main/java/com/gutenberg/MainActivity.java

```diff
@@ -12,4 +19,33 @@
     protected String getMainComponentName() {
         return "gutenberg";
     }
+
+    private RNReactNativeGutenbergBridgeModule module;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((ReactApplication) getApplication()).getReactNativeHost()
+                .getReactInstanceManager()
+                .addReactInstanceEventListener(context -> module = context.getNativeModule(RNReactNativeGutenbergBridgeModule.class));
+    }
+
+    @Override
+    public boolean onKeyUp(int keyCode, KeyEvent event) {
```

Comment:
I wonder if we will need to add a debouncer here. Just trying to toggle bold formatting makes it toggle twice or more for me (tested on an iOS Simulator).
Also and more worrying is when I keep the modifier key (so command on macos) pressed it keeps repeating the key over and over. so for `cmd+b` it keeps on toggling the selected text in bold, and for `cmd+v` it keeps on pasting the copied text over and over until I release the `cmd` key.
I'll look into it a bit, but maybe you have an idea @kmdupr33 ?

Reply:
Just tested in `develop` and it looks like a regression, at least on iOS

Reply:
Interesting. Looking into this now.

Reply:
Ok. This is resolved 👍